lib/action_smser/base.rb

```diff
@@ -120,7 +120,7 @@ def to_numbers_array
     else
       [@to.to_s]
     end
-    array.collect{|number| number.gsub(/^(\+|0)/, "")}
+    array.collect{|number| number.gsub(/^(\+|0{1,2})/, "")}
```

Comment:
I assume the intention here was to strip both `0` and `00` - correct? cause `+` and `00` are the international format prefixes. so I've added `00` to the mix and a test further below.